packages/react-search-ui-views/src/SearchBox.js

```diff
@@ -15,32 +25,48 @@ function SearchBox(props) {
     isFocused,
     inputProps,
     onChange,
-    onSelectAutocomplete,
     onSubmit,
     value
   } = props;
   const focusedClass = isFocused ? "focus" : "";
 
-  return (
-    <form onSubmit={onSubmit}>
-      <Downshift
-        inputValue={value}
-        onChange={onSelectAutocomplete}
-        onInputValueChange={onChange}
-        itemToString={item =>
-          // TODO
-          item
-            ? item[autocompleteResults.titleField]
-              ? item[autocompleteResults.titleField].raw ||
-                item[autocompleteResults.titleField].snippet
-              : item.suggestion
-            : ""
+  const onSelectAutocomplete =
+    props.onSelectAutocomplete ||
+    (selection => {
+      if (!selection.suggestion) {
+        const url = selection[autocompleteResults.urlField]
+          ? selection[autocompleteResults.urlField].raw
+          : "";
+        if (url) {
+          const target = autocompleteResults.linkTarget || "_self";
+          window.open(url, target);
         }
-      >
-        {({ getInputProps, getItemProps, getMenuProps, isOpen }) => {
-          let index = 0;
-          let autocompleteClass = isOpen === true ? " autocomplete" : "";
-          return (
+      }
+    });
+
+  return (
+    <Downshift
+      inputValue={value}
+      onChange={onSelectAutocomplete}
+      onInputValueChange={onChange}
+      stateReducer={(state, changes) => {
```

Comment:
This could be simplified:
```
stateReducer={(_, changes) => changes}
```

Reply:
Good catch. That was actually not needed, I just forgot to remove it, so I just removed it.